components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/APIManagerConfiguration.java

```diff
@@ -2052,6 +2066,10 @@ public static Map<String, String> getPersistenceProperties() {
         return persistenceProperties;
     }
 
+    public static Map<String, String> getOperationalPolicyProperties() {
+        return operationPolicyProperties;
```

Comment:
Should this be modifiable after initially populating this map in L373-L385? If not, how about keeping it as an immutable map?

Reply:
No it is not modifiable. 
However, I think we have followed the same practice in other places as well. We may fix other places as well in a different effort.